wemake_python_styleguide/visitors/ast/classes.py

```diff
@@ -158,11 +160,62 @@ def _check_bound_methods(self, node: types.AnyFunctionDef) -> None:
                     oop.AsyncMagicMethodViolation(node, text=node.name),
                 )
 
+        self._check_useless_overwritten_methods(
+            node,
+            class_name=node_context.name,
+        )
+
     def _check_method_contents(self, node: types.AnyFunctionDef) -> None:
         if node.name == constants.INIT:
             if walk.is_contained(node, self._not_appropriate_for_init):
                 self.add_violation(bp.YieldInsideInitViolation(node))
 
+    def _get_call_stmt_of_useless_method(
+        self,
+        node: types.AnyFunctionDef,
+    ) -> Optional[ast.Call]:
+        # consider next body as possible candidate of useless method:
+        # 1) Optional[docstring]
```

Comment:
Do you mean that in case the method has docstring and everything else is as we discussed then it is not useless?
```python
class Test(SomeBase):
    """Test class."""

    def function(self, argument):
        """
        Some method.
        """
        return super(Test, self).function(argument)
```

Reply:
No, the one you provide is useless. I was talking about methods with just docs. Like this one: https://github.com/wemake-services/wemake-python-styleguide/blob/d08cbd6ef2c416bdf69bb470984858a6040a9c6d/wemake_python_styleguide/visitors/base.py#L131

Reply:
here I meant that second item is required and first item with docstring is optional